config/initializers/rack_attack.rb

```diff
@@ -12,9 +12,22 @@ def self.protected_path?(request)
     PROTECTED_ROUTES.any? { |route| request.path.start_with?(route) }
   end
 
+  def self.api_request?(request)
+    request.path.starts_with?("/api/")
+  end
+
+  def self.auth_token(request)
+    request.get_header("HTTP_AUTHORIZATION")
+  end
+
   throttle("protected routes (OTP)", limit: 5, period: 20.seconds) do |request|
     request.ip if protected_path?(request)
   end
+
+  # Throttle /api requests by auth token (1000 requests per 5 minutes)
+  throttle("API requests by auth token", limit: 1000, period: 5.minutes) do |request|
+    auth_token(request) if api_request?(request) && !protected_path?(request)
```

Comment:
Do we need the `&& !protected_path?(request)`?

Not really sure what it's guarding against.

Reply:
We did something similar with NPQ, we probably don't need this for rect, i'll remove it.

Reply:
Ah yeah I see now on NPQ - having different rate limit rules for `/api/guidance`.